docs/guides/generating-uht-compatible-headers.md

```diff
@@ -2,14 +2,14 @@
 
 ## Supported versions
 
-While the UHT header generator is only officially supported in `4.25+`, it has worked for older game versions (tested on `4.18.3`; `4.17` (has some default property issues that should be fixed soon)).
+While the UHT header generator is only officially supported in `4.25+`, it has worked for older game versions (tested on `4.18.3`; `4.17` (has some default property issues that should be fixed soon)). It also works for `5.0+`.
 
 ## How to use
 
 The key bind to generate headers is by default `CTRL` + `Numpad 9`, and it can be changed in `Mods/Keybinds/Scripts/main.lua`.
 
 To utilize the generated headers to their full potential, see [UE4GameProjectGenerator](https://github.com/Buckminsterfullerene02/UE4GameProjectGenerator) by Archengius (link to Buck's fork because of a couple fixes that Arch is too lazy to merge).
-> The project generator only works in `>4.25`. If generating a project for an older engine version, generate it by compiling the project generator for `4.25+`.
+> The project generator is only guaranteed to work in `>4.22`, though it has worked for some older versions. If generating a project for an older engine version than `4.25`, generate it by compiling the project generator for `4.25+` first.
```

Comment:
Can we change the wording here ?
We are in fact not guaranteeing that the project generator works on any engine version, we have merely tested that it works on certain versions.

Reply:
Dyslexia moment